specification/personal-demographics.yaml

```diff
@@ -489,19 +497,21 @@ paths:
+        | Basic search                            | `family`=`Smith`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`                                        | HTTP Status 200 with single search result.                              |
+        | Wildcard search                         | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`                                                                                                   | HTTP Status 200 with search result of two.                              |
+        | Wildcard search with email and phone    | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`                                          | HTTP Status 200 with single search result.                              |
+        | Search with limited results             | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`, `_max-results`=`1`                      | HTTP Status 200 with single search result.                              |
+        | Search with date range                  | `family`=`Smith`, `gender`=`female`, `birthdate`=`ge2010-10-21`, `birthdate`=`le2010-10-23`, `email`=`jane.smith@example.com`, `phone`=`01632960587`            | HTTP Status 200 with single search result.                              |
+        | Fuzzy search                            | `family`=`Smith`, `given`=`Jane`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`, `_fuzzy-match`=`true` | HTTP Status 200 with single search result.                              |
+        | Restricted patient search               | `family`=`Smythe`, `given`=`Janet`, `gender`=`female`, `birthdate`=`eq2005-06-16`, `email`=`jane.smith@example.com`, `phone`=`01632960587`                      | HTTP Status 200 with single search result with restricted data omitted. |
+        | Compound name search                    | `family`=`Smith`, `given`=`John Paul`, `given`=`James`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`  | HTTP Status 200 with single search result.                              |
+        | Unsuccessful search                     | Any other valid search query parameters                                                                                                                         | HTTP Status 200 with no search results                                  |
+        | Unsuccessful search on email and phone  | `email`=`j.smith@example.com`, `phone`=`0163`                                                                                                                   | HTTP Status 200 with no search results                                  |
```

Comment:
Maybe add the following?:
HTTP Status 501 with a mixture of valid/invalid search criteria | `family`=`Sm*`, `gender`=`female`, 'invalidParam'='123', `birthdate`=`eq2010-10-22`,  `email`=`j.smith@example.com`, `phone`=`0163` | HTTP Status 501 with problem description